services/web/server/src/simcore_service_webserver/exporter/formatters/sds/sds.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from simcore_service_webserver.exporter.formatters.sds.text_files import (
+    write_text_files,
+)
+from simcore_service_webserver.exporter.formatters.sds.xlsx import write_xlsx_files
```

Comment:
STYLE: We use relative paths in osparc

Reply:
Why not adding this file directly in the ``__init__``?

Reply:
long story short: had to use full paths during testing to speed up development. Will change to relative.